### PR TITLE
Added ref to stop name.

### PR DESCRIPTION
**Pull request by luktar**

### CODE REVIEW

File: src/utilities/mapUtilities.js [hunk 1]
```diff
@@ -10,24 +10,28 @@ const generateConnectionData = (connection, tileId) => {
...
-    return `${id}`;
+const generateStopName = stop => {
+  let result = stop.name || stop.id;
+  if (stop.number) {
+    result += ' ' + stop.number;
   }
+  if (stop.stopType === 1) return result;
+
+  const refTag = stop.tags.find(x => x.key === 'ref');
+  if (refTag) return (result += `, ref: ${refTag.value || '-'}`);
```
Comment (AdamSarach) [inline src/utilities/mapUtilities.js]: return (result + `, ref: ${reftag?.value || '-'}`);
?

File: src/utilities/mapUtilities.js [hunk 2]
```diff
@@ -10,24 +10,28 @@ const generateConnectionData = (connection, tileId) => {
...
-  } else {
-    return `${id}`;
+const generateStopName = stop => {
+  let result = stop.name || stop.id;
+  if (stop.number) {
+    result += ' ' + stop.number;
   }
+  if (stop.stopType === 1) return result;
+
+  const refTag = stop.tags.find(x => x.key === 'ref');
```
Comment (AdamSarach) [inline src/utilities/mapUtilities.js:32]: imo it's more readable when you name item you iter by: (tagObject => tagObject.key === 'ref')

File: src/utilities/mapUtilities.js [hunk 3]
```diff
@@ -10,24 +10,28 @@ const generateConnectionData = (connection, tileId) => {
     return {
       osmStopId: connection[0].id.toString(),
       gtfsStopId: connection[1].id.toString(),
-      tileId: tileId
+      tileId: tileId,
     };
   } else if (connection[0].isOsm === false && connection[1].isOsm === true) {
```
Comment (AdamSarach) [inline src/utilities/mapUtilities.js]: } else if (!(connection[0].isOsm)  && connection[1].isOsm ) {
?

**Review by AdamSarach — APPROVED** [review]
I do not like these `+=` (looks weird) but im note sure whether using `string.concat()` or anything else would satisfy me more :P

File: src/utilities/mapUtilities.js (hunk 3 above)
Comment (luktar) [reply src/utilities/mapUtilities.js]: To się poprawi przy okazji refaktoringu.

File: src/utilities/mapUtilities.js (hunk 2 above)
Comment (luktar) [reply src/utilities/mapUtilities.js:32]: In my opinion, it's less readable when this name is long. You should be more careful about proper list naming than list elements. You can call it an `item` but it'll always be a member of the list which is a simple value or an object. So if the list is called `tags` element will always be `tag`. 
The `Object` suffix is not necessary.
A long name can also cause overloading a code with that name. It happens when you use a name several times in an inline function.

File: src/utilities/mapUtilities.js (hunk 1 above)
Comment (luktar) [reply src/utilities/mapUtilities.js]: Ok

**Pull request merged**